<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_ChoixProfil</class>
 <widget class="QDialog" name="f_ChoixProfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Lb_MdP">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>85</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_MdP">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>85</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_Profil">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>25</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Profil :</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="actionValider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>145</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="Cb_Bx_Profil">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
